blockchain/fullblocktests/generate.go

```diff
@@ -1044,7 +1044,6 @@ func Generate() (tests [][]TestInstance, err error) {
 	//                \-> b38(10)
 	//
 	// OP_CHECKMULTISIG counts for 20 sigops.
-	tooManySigOps = repeatOpcode(txscript.OP_CHECKMULTISIG, maxBlockSigOps/20)
```

Comment:
It looks like a merge artifact (yay merges...).  Notice how this line is 100% the same as `manySigOps` above.  Either this line can be removed like this, or it can be kept and the line below updated to append to `tooManySigOps`.

That said, I sort of think I'd prefer the latter since it won't break the test if the `manySigOps` above is changed in the future.